examples/image_classifier/streaming_video_client_side_batching/custom_handler.py

```diff
@@ -0,0 +1,48 @@
+import base64
+import io
+
+import torch
+import torch.nn.functional as F
+from PIL import Image
+
+from ts.torch_handler.image_classifier import ImageClassifier
+from ts.utils.util import map_class_to_label
+
+
+class ModelHandler(ImageClassifier):
+    def __init__(self):
+        super(ModelHandler, self).__init__()
+
+    def preprocess(self, data):
+
+        images = []
+        for row in data:
+            imgs = row.get("data") or row.get("body")
+            imgs = list(imgs.values())
+            for image in imgs:
```

Comment:
are we getting only one array here, as a result of framed being batched on the client side?

Reply:
Yes, we are sending  the frames as a dict in a  json payload.